client/common/cmdline.c

```diff
@@ -3629,7 +3629,6 @@ BOOL freerdp_client_load_addins(rdpChannels* channels, rdpSettings* settings)
 		{ FreeRDP_EncomspVirtualChannel, ENCOMSP_SVC_CHANNEL_NAME, settings },
 #endif
 		{ FreeRDP_RemdeskVirtualChannel, REMDESK_SVC_CHANNEL_NAME, settings },
-		{ FreeRDP_RDP2TCPArgs, RDP2TCP_DVC_CHANNEL_NAME, settings->RDP2TCPArgs },
```

Comment:
Why not just fix this here? `{ 0, RDP2TCP_DVC_CHANNEL_NAME, settings->RDP2TCPArgs },` should work or not?

Reply:
Well if I do this the rdp2tcp channel will always be loaded even with NULL settings->RDP2TCPArgs